src/VCAuthn/Controllers/PresentationRequestController.cs

```diff
@@ -50,6 +58,58 @@ public async Task<ActionResult> Poll([FromQuery(Name = IdentityConstants.Challen
             return Ok();
         }
 
+        [HttpGet(IdentityConstants.AuthorizeCallbackUri)]
+        public async Task<ActionResult> AuthorizeCallback([FromQuery(Name = IdentityConstants.ChallengeIdQueryParameterName)] string sessionId)
```

Comment:
Perhaps we should make this into an `AuthorizeCallbackEndpoint` instead of one broken out at a controller level?

Reply:
👍